rpc/README.md

```diff
@@ -0,0 +1,96 @@
+# RPC
```

Comment:
minor nit: let's change this to "Miden node RPC".

Reply:
@bobbinth shell I change RPC to Miden node RPC in the whole document or only in the headline?

Reply:
I think just in the headline is fine.